docs/pages/branding/design-kits.tsx

```diff
@@ -0,0 +1,36 @@
+import * as React from 'react';
+import Head from 'docs/src/modules/components/Head';
+import ThemeProvider from 'docs/src/modules/ThemeContext';
+import CssBaseline from '@material-ui/core/CssBaseline';
+import Divider from '@material-ui/core/Divider';
+import AppHeader from 'docs/src/layouts/AppHeader';
+import AppFooter from 'docs/src/layouts/AppFooter';
+import DesignKitHero from 'docs/src/components/productDesignKit/DesignKitHero';
+import DesignKitValues from 'docs/src/components/productDesignKit/DesignKitValues';
+import DesignKitDemo from 'docs/src/components/productDesignKit/DesignKitDemo';
+import DesignKitFAQ from 'docs/src/components/productDesignKit/DesignKitFAQ';
+import Testimonials from 'docs/src/components/home/Testimonials';
+import HeroEnd from 'docs/src/components/home/HeroEnd';
+
+export default function Home() {
+  return (
+    <ThemeProvider>
+      <Head
+        title="MUI in your favorite design tool."
+        description="For the designers out there, pick your favorite design tool to enjoy our components. Get the consistency right when working with developers."
+      />
+      <CssBaseline />
+      <AppHeader />
+      <main>
+        <DesignKitHero />
+        <DesignKitValues />
+        <DesignKitDemo />
+        <DesignKitFAQ />
+        <Testimonials mode="dark" />
```

Comment:
what do you mean by `inconsistent`? I think it is clear that this Testimonials will appear as dark regardless of the user's prefers-scheme. Is the prop name that is not clear?

Reply:
"inconsistent" because the mode is a user choice. If we force something different it's no longer a mode. In other words: Why does the UI look so different between light mode (alternating mode) compared to dark mode (no alternating).

Reply:
so, is it better to do this instead?.

```js
<ThemeProvider theme={darkTheme}>
  <Testimonials />
</ThemeProvider>
```

> Why does the UI look so different between light mode (alternating mode) compared to dark mode (no alternating)

In this context, I think to give a clear separation between section. I don't think it is weird but will let @danilo-leal express his thought.

Reply:
It shouldn't use "mode" at all since it's clearly not about the mode the user wants their UI in. This forces a specific color that incidentally matches dark mode. But why this is force, is unclear.

Reply:
So, you are saying the design should not have dark background on light mode? I am trying to understand your point about `It shouldn't use "mode" at all`, whether it is the design or technical implementation.

Reply:
This section is mode invariant.

Reply:
That is not what a mode is. You may think "mode invariant" sounds smart but it's not that at all. You're not rendering a mode here. It just incidentally looks that way.

Reply:
Initially, this section was rendered on most pages where it was used in "dark mode" regardless of user choice. I had it in a light mode only on the Pricing page, so that is probably why Jun is forcing the dark mode on these pages. And what I think Olivier tried to say is exactly that: in most places, it is rendered with a look that matches the dark mode, so in those, the mode is irrelevant (or invariant, just a way to say).

I used this discussion [to standardize this section](https://www.figma.com/file/4uv2dT18rXJPZBbrbpw61q/Marketing-site?node-id=2837%3A13167) as "always dark" as in it doesn't change regardless of the user mode choice. So we can probably remove the `mode="dark"` there?!

Reply:
will create a fix in another PR since, I don't think it is related to these 2 pages.